bertopic/backend/_utils.py

```diff
@@ -2,11 +2,13 @@
 
 # Imports for light-weight variant of BERTopic
 from bertopic.backend._sklearn import SklearnEmbedder
+from bertopic._utils import MyLogger
 from sklearn.pipeline import make_pipeline
 from sklearn.decomposition import TruncatedSVD
 from sklearn.feature_extraction.text import TfidfVectorizer
 from sklearn.pipeline import Pipeline as ScikitPipeline
 
+logger = MyLogger("WARNING")
```

Comment:
This might create duplicates of the logger but I am not entirely sure. I believe this might be related: https://github.com/MaartenGr/BERTopic/pull/1894#discussion_r1574244162

Reply:
I think as a comment there says, instantiating MyLogger will return a reference to the same logger instance each time. Although setting the log level here will affect all instances of MyLogger (because they're the same underlying logger), because it's also in the `__init__()` of `BERTopic` it should then set it correctly. But the suggestions to clean this up in that PR sound sensible, I'm happy to adopt those once it's merged.

Reply:
Oh but then I've added setting the level in `select_backend()` because I didn't realise the implications of `MyLogger` at that point. I think I can just remove the line I added to do that, and it'll be fine because the log level is already set.

Reply:
For now this is fine. I might update the logger at some point but that will be separate from this PR.